spec/martian_robot_spec.rb

```diff
@@ -0,0 +1,85 @@
+$LOAD_PATH << File.expand_path('. ./lib', __FILE__)
+# require 'martian_robots'
+require 'martian_robot'
+
+RSpec.describe MartianRobot do
+
+    let (:robot) {MartianRobot.new(@coordinates, @orientation)}
+    # let(:martian_robots) {MartianRobots.new}
```

Comment:
It's always best to delete any commented out code before submitting a pull request.